app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    _, current_file, new_file_path = command.split()
+    dir_list = new_file_path.split("/")
+    new_file_name = dir_list.pop()
+    new_dir = ""
+
+    for directory in dir_list:
```

Comment:
you don't really need a for loop here since `makedirs` will create all necessary dirs included in destination path if they don't exist